front/lib/api/assistant/actions/dust_app_run.ts

```diff
@@ -315,6 +315,7 @@ export class DustAppRunConfigurationServerRunner extends BaseActionConfiguration
     // As we run the app (using a system API key here), we do force using the workspace credentials so
     // that the app executes in the exact same conditions in which they were developed.
     const runRes = await api.runAppStreamed(
+      auth.getNonNullableUser(),
```

Comment:
Same, we likely don't want to throw here ?

Reply:
I checked and this code path is only used with a valid authentication that bundles a user. So should be fine.